lib/constants/dimensions.dart

```diff
@@ -43,6 +43,12 @@ class Dimensions {
   static const double mentorCardWidth = 260;
   static const double chipVisualDensity = -4;
   static const double chipMaxWidth = 100;
+  static const double loginBoxWidth = 360;
+  static const double lineHeight = 1;
+  static const double textBoxWidth = 200;
+  static const double sizedBoxWidth = 360;
+  static const double iconWidth = 20;
+  static const double iconSpaceWidth = 10;
```

Comment:
Whenever possible, widths and heights should be multiples of 8 (preferred), or multiple of 4 (for smaller components). This so that elements align nicely in an imaginary grid. Check out this for more details: https://m2.material.io/design/layout/spacing-methods.html

Reply:
Ah, fantastic. This is a great read. I'll read up more on material design, and make these changes.